modules/cluster-logging-manual-rollout-rolling.adoc

```diff
@@ -126,9 +156,14 @@ If you performed a rollout on the Elasticsearch pod you used in the previous com
 +
 For example:
 +
+[source,terminal]
 ----
 $ oc exec elasticsearch-cdm-5ceex6ts-1-dcd6c4c7c-jpw6 -c elasticsearch -- es_util --query=_cluster/health?pretty=true
-
+----
++
+.Example output
+[source,terminal]
+----
```

Comment:
Interesting; Looks like this query returns JSON. This could be [source,json] as well.